eventstore/model.go

```diff
@@ -130,7 +134,13 @@ func (m *Model) Reduce(event core.Event) error {
 }
 
 func (m *Model) validInstance(v interface{}) (bool, error) {
-	vLoader := gojsonschema.NewGoLoader(v)
+	var vLoader gojsonschema.JSONLoader
+	if m.jsonMode {
```

Comment:
If we are in json mode.. use a bytes loader instead of a goloader. Which basically means use plain json vs a go type.